main/system.h

```diff
@@ -83,15 +77,17 @@
 #endif
 
 #if defined(IS_ATHENA)
-#define IRIS_CONFIG_SPI spiREG1
-#define IRIS_SPI spiREG3
-#define SBAND_SPI spiREG2
-#define SD_SPI spiREG3
+
+    #define IRIS_CONFIG_SPI spiREG4 //SPI1
+    #define IRIS_SPI spiREG5        //SPI3
+    #define SBAND_SPI spiREG3       //SPI2
+    #define SD_SPI spiREG1          //?
 #else // These values are expected to be adjusted based on what the developer is working on
-#define IRIS_CONFIG_SPI spiREG1
-#define IRIS_SPI spiREG4
-#define SBAND_SPI spiREG2
-#define SD_SPI spiREG3
+    #define IRIS_CONFIG_SPI spiREG4 //SPI1
+    #define IRIS_SPI spiREG5        //SPI3
+    #define SBAND_SPI spiREG3       //SPI2
+    #define SD_SPI spiREG1          //?
```

Comment:
Did you change these knowing they are likely to get all messed up by development? I don't think that's necessarily a bad thing